rasa_core/run.py

```diff
@@ -106,17 +109,22 @@ def _create_external_channel(channel, port, credentials_file):
         input_blueprint = SlackInput(
             credentials.get("slack_token"),
             credentials.get("slack_channel"))
+    elif channel == "telegram":
+        input_blueprint = TelegramInput(
+            credentials.get("access_token"),
+            credentials.get("verify"),
+            credentials.get("webhook_url"))
     else:
-        Exception("This script currently only supports the facebook "
-                  "and slack connectors.")
+        Exception("This script currently only supports the facebook,"
+                  ", telegram and slack connectors.")
 
     return HttpInputChannel(port, None, input_blueprint)
 
 
 def create_input_channel(channel, port, credentials_file):
     """Instantiate the chosen input channel."""
 
-    if channel == "facebook" or channel == "slack":
+    if channel == "facebook" or "slack" or "telegram":
```

Comment:
this will always be true. how about

`if channel in ['facebook', 'slack', 'telegram']:`

Reply:
I think I already fixed that once for slack? why is it back in here?

Reply:
maybe this was branched from the slack branch before you spotted that

Reply:
you did not, but I will fix it :D

Reply:
mhm I did, it is fixed on master :D https://github.com/RasaHQ/rasa_core/blob/master/rasa_core/run.py#L119

Reply:
but yeah, good spot :+1:

Reply:
oh yeah sorry, I think I must've deleted the wrong one when resolving conflicts